ttkwidgets/frames/toggledframe.py

```diff
@@ -2,13 +2,18 @@
 Author: RedFantom
 License: GNU GPLv3
 Source: This repository
+
+Improved by rdbende
 """
+
 import tkinter as tk
+from pathlib import Path
 from tkinter import ttk
-import os
-from PIL import Image, ImageTk
```

Comment:
Apparently even Python 3.6.8 on macOS uses Tk 8.6 these days, so indeed for the PNG files `pillow` is no longer necessary 🥳